### DISCUSSION TITLE
Stub catalog item

### DISCUSSION THREAD

**Discussion by imakihi** [TerriaJS/terriajs#6136] [Q&A] [answered]
Hi community.
This is a very simple and silly question but what is the stub and what kind of data stubcatalogitem can deal with?
https://docs-v8.terria.io/guide/connecting-to-data/catalog-type-details/stub/

I have to write up some documents about Terria in Japanese and I couldn't find any info regarding the stub...

Thank you.

Hiroo

**Comment by zoran995** [comment, marked as answer]
It's used internally when received definition can't be parsed, top comment in file [StubCatalogItem.ts](https://github.com/TerriaJS/terriajs/blob/main/lib/Models/Catalog/CatalogItems/StubCatalogItem.ts). So when the stub is created there will be info for the user that something is not defined correctly. 
Maybe it shouldn't be mentioned in the doc as catalog item user can create

**Reply by imakihi** [reply]
@zoran995 Thank you for the explanation. I think I am going to remove this item from my Japanese doc.
Thank you again.
Hiroo
